internal/flowcontrol/base_flow_controller.go

```diff
@@ -99,7 +99,7 @@ func (c *baseFlowController) maybeAdjustWindowIncrement() {
 
 	timeSinceLastWindowUpdate := time.Since(c.lastWindowUpdateTime)
 	// interval between the window updates is sufficiently large, no need to increase the increment
-	if timeSinceLastWindowUpdate >= 2*rtt {
+	if timeSinceLastWindowUpdate >= 4*protocol.WindowUpdateThreshold*rtt {
```

Comment:
Nit: Magic number here ;)

Reply:
Yes, and this magic number comes from here: https://docs.google.com/document/d/1F2YfdDXKpy20WVKJueEf4abn_LVZHhMUMS5gX6Pgjl4/edit#heading=h.o1yqsbo737tc.

What is your suggestion?